packages/components/src/components/icon/common/ic-document-number-verification.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="12" viewBox="0 0 16 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You need to run SVGO to minify the files, look into the README file for more details.